test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -85,19 +91,35 @@ def test_dlc_version_dockerfiles(image):
                 if "example" not in dockerfile_path and f"{os.sep}{fw_version_major_minor}" in dockerfile_path:
                     dockerfiles.append(dockerfile_path)
 
+    # For the collected dockerfiles above, note the DLC major versions in each Dockerfile if python version matches
+    # the current image under test
     versions = {}
     dlc_label_regex = re.compile(r'LABEL dlc_major_version="(\d+)"')
+    python_version_regex = re.compile(r'ARG PYTHON_VERSION=(\d+.\d+)')
```

Comment:
nit: It doesn't matter in this particular case, because our definition of python version always has the format `X.Y.Z`, but when you have a regular expression, it is best to use `r"\d+\.\d+"` rather than `r"\d+.\d+"` to specify that the `.` is meant to be matched literally, and not as "any-character".